app/views/collection_resources/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :before_content do %>
+  <%= govuk_back_link href: collection_resources_path %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @collection_resource, url: collection_resources_path, method: :post do |f| %>
+      <%= f.hidden_field :year %>
+      <%= f.hidden_field :log_type %>
+      <%= f.hidden_field :mandatory, value: false %>
+
+      <%= f.govuk_error_summary %>
+
+      <span class="govuk-caption-l"><%= "#{@collection_resource.log_type.humanize} #{text_year_range_format(@collection_resource.year)}" %></span>
+      <h1 class="govuk-heading-l">Add a new collection resource</h1>
+
+      <p class="govuk-body">
+        This file will be available for all users to download.
+      </p>
+
+      <%= f.govuk_file_field :file,
+        label: { text: "Upload file", size: "m" } %>
+
+      <%= f.govuk_text_field :short_display_name,
+                      label: nil,
+                      hint: { text: safe_join(["This will be used in the download link on the homepage. Do not include the log type or collection year.",
+                                               content_tag(:br),
+                                               "For example, if you enter “bulk upload change log”, the download link will say “Download the #{@collection_resource.log_type} bulk upload change log (#{text_year_range_format(@collection_resource.year)})”."]) } %>
```

Comment:
Could you add one more <br>? On the edit page too. Separate out the sentences more clearly, similar to the designs.

Reply:
This is what it looks like atm, the description screenshot it a bit outdated. What do you think?
<img width="803" alt="image" src="https://github.com/user-attachments/assets/5d75dcbb-e51d-4128-aedc-560002116c48">

Reply:
Okay that's all good